commit 10e6492ca5373eed3a62bfad38565d254bd03d9b
Author: Justin Maggard <jmaggard10@gmail.com>
Date:   Fri Feb 7 09:23:10 2014 -0800

    Revert "btrfs-progs: mkfs: enable extended refs by default"
    
    We automatically turn on extended refs as needed in kernel.
    
    This reverts commit 9f0c53f574b242b0d5988db2972c8aac77ef35a9.

diff --git a/mkfs.c b/mkfs.c
index 33369f9..fbadcde 100644
--- a/mkfs.c
+++ b/mkfs.c
@@ -46,8 +46,6 @@
 
 static u64 index_cnt = 2;
 
-#define DEFAULT_MKFS_FEATURES	(BTRFS_FEATURE_INCOMPAT_EXTENDED_IREF)
-
 #define DEFAULT_MKFS_LEAF_SIZE 16384
 
 struct directory_name_entry {
@@ -1157,15 +1155,10 @@ static void list_all_fs_features(void)
 
 	fprintf(stderr, "Filesystem features available at mkfs time:\n");
 	for (i = 0; i < ARRAY_SIZE(mkfs_features) - 1; i++) {
-		char *is_default = "";
-
-		if (mkfs_features[i].flag & DEFAULT_MKFS_FEATURES)
-			is_default = ", default";
-		fprintf(stderr, "%-20s- %s (0x%llx%s)\n",
+		fprintf(stderr, "%-20s- %s (0x%llx)\n",
 				mkfs_features[i].name,
 				mkfs_features[i].desc,
-				mkfs_features[i].flag,
-				is_default);
+				mkfs_features[i].flag);
 	}
 }
 
@@ -1260,7 +1253,7 @@ int main(int ac, char **av)
 	int dev_cnt = 0;
 	int saved_optind;
 	char estr[100];
-	u64 features = DEFAULT_MKFS_FEATURES;
+	u64 features = 0;
 
 	while(1) {
 		int c;
